### ISSUE TITLE
text2text record validate not storing annotation

### ISSUE THREAD

**Issue by dvsrepo** [recognai/rubrix#1724]
**Describe the bug**
Validate action fails on text2text manual labeling mode.

Apparently, even if we see a prediction and we click `validate` the `annotation` field is not correctly populated (text is missing, only score).

The error on the server is:

```js
code: "rubrix.api.errors::ValidationError"
params: {model: "Request",…}
errors: [{loc: ["body", "records", 0, "annotation", "sentences", 0, "text"], msg: "field required",…}]
0: {loc: ["body", "records", 0, "annotation", "sentences", 0, "text"], msg: "field required",…}
loc: ["body", "records", 0, "annotation", "sentences", 0, "text"]
0: "body"
1: "records"
2: 0
3: "annotation"
4: "sentences"
5: 0
6: "text"
msg: "field required"
type: "value_error.missing"
model: "Request"
```

Payload:
```json
{
  "records": [
    {
      "id": 12,
      "metadata": {
        "length": "58",
        "model": "Helsinki-NLP/opus-mt-en-es",
        "dataset": "europarl_bilingual"
      },
      "prediction": {
        "agent": "Helsinki-NLP/opus-mt-en-es",
        "sentences": [
          {
            "text": "Si la Asamblea está de acuerdo, haré lo que ha sugerido el Sr. Evans.",
            "score": 0.6535725617426158
          }
        ]
      },
      "annotation": {
        "sentences": [
          {
            "score": 1
          }
        ],
        "agent": "recognai"
      },
      "status": "Validated",
      "selected": false,
      "search_keywords": [],
      "text": "If the House agrees, I shall do as Mr Evans has suggested."
    }
  ]
}
```

**To Reproduce**
1. Go to text2text dataset
2. Click on  hand labelling
4. validate one record where you see a prediction
5. See error popup

**Expected behavior**
Annotation should be properly saved with the selected prediction

**Screenshots**
<img width="1521" alt="Screenshot 2022-09-20 at 12 34 54" src="https://user-images.githubusercontent.com/1107111/191236955-6c6eb111-ce7f-4a4c-af51-762715d412dc.png">

<img width="1642" alt="Screenshot 2022-09-20 at 12 35 03" src="https://user-images.githubusercontent.com/1107111/191236951-460b30af-0d4a-4a81-85a1-f524b52c8af4.png">

**Environment (please complete the following information):**
 - OS [e.g. iOS]:
 - Browser [e.g. chrome, safari]:
 - Rubrix Version dev
 - ElasticSearch Version [e.g. 7.10.2]:
 - Docker Image (optional) [e.g. rubrix:v0.10.0]:

**Additional context**

**Comment by frascuchon** [comment]
Is this issue still happening @leiyre @dvsrepo?

**Comment by leiyre** [comment]
@frascuchon when annotation exists the behavior is ok, I would like to have a dataset with prediction and without annotation for testing

**Comment by frascuchon** [comment]
The dataset `reclamaciones_fine_tuned-no-annotations` is already created in PRE.

**Comment by leiyre** [comment]
Seems to have been fixed in #1813

**Comment by frascuchon** [comment]
@dvsrepo can you reproduce this issue with latest Argilla version?
